tools/force_python_version.py

```diff
@@ -0,0 +1,43 @@
+'''
```

Comment:
how about calling this just `python_version`, or `python_version_handling` or something like that, since it also provides utilities to check it etc.?